onnxruntime/core/optimizer/qdq_transformer/ensure_unique_dq_for_node_unit.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+
+namespace onnxruntime {
+
+/**
+ * Graph transformer that duplicates DQ nodes in order to ensure that each potential QDQ node unit has unique DQ nodes
```

Comment:
What are the performance implications of this change? How did it (not) work before?